### DISCUSSION TITLE
Ideas Collection

### DISCUSSION THREAD

**Discussion by xinyao27** [xinyao27/next-devtools#15] [Ideas]
If you have any ideas/requests for features builtin into Next DevTools, feel free to brainstorm your ideas below!

**Comment by RodrigoTomeES** [comment]
Maybe it would be fine to include @next/bundle-analyzer or @welldone-software/why-did-you-render inside next-devtools 🤔

**Reply by xinyao27** [reply]
Great, let me study how to integrate it into next devtools.

**Comment by Gawdfrey** [comment]
Big fan of the devtools!
Would love to have some sort of tree view of the layout showing where the client and server boundaries are. I've seen some VSCode plugins do this inefficiently, but would be much better suited here. Will try to find some examples.

**Reply by Gawdfrey** [reply]
https://www.next-nav.com/ One of the plugins I've seen!
